notebook/analysis/benchmark_results.ipynb

```diff
@@ -10517,12 +10517,1360 @@
     "plot"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sex Prediction"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sample Split Positive Control"
```

Comment:
Are these results fairly comparable to what they found in the paper you linked, when they did prediction from gene expression? Or are the experiments you're doing substantially different so you can't compare?

Reply:
Yeah, both of our accuracies top out around 90%. (Which is interesting in and of itself because they're using only X and Y chromosome genes)